src/main/java/io/cryostat/recordings/Recordings.java

```diff
@@ -1087,14 +1092,16 @@ public record LinkedRecordingDescriptor(
 
     // TODO include jvmId and filename
     public record ArchivedRecording(
-            String name,
+            String jvmId,
+            String filename,
```

Comment:
I would prefer this stays as `name` rather than `filename`, since this field gets serialized in API responses and WebSocket notifications, so the naming is actually part of the exposed API. I'm pretty sure there are bits of code in the web-client that expect to find the field as `name`, for example.